### PR TITLE
[1LP][RFR] No more session fixtures

### PR DESCRIPTION
**Pull request by izapolsk**
there is a plan to implement appliance markers and generate appliances based on those markers. 
We should get rid of session scoped fixuters using appliance fixture. This PR is one of steps we have to undertake to achieve above.

based on https://github.com/ManageIQ/integration_tests/pull/9638

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 1783bffc252d1c476495d524dd5d318c655af6a1:

`cfme/fixtures/browser.py`:
- :red_circle: Line 4:1: **F401** *'pytest' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/fixtures/cfme_data.py [hunk 1]
```diff
@@ -4,5 +4,5 @@
 
 
 @pytest.fixture(scope="session")
```
Comment (jawatts) [inline cfme/fixtures/cfme_data.py]: is this suppose to be session still?

Comment (izapolsk) [reply cfme/fixtures/cfme_data.py]: yes. I have to get rid of session scoped fixtures which use appliance. the rest can be left as is.

File: cfme/fixtures/cfme_data.py [hunk 2]
```diff
@@ -4,5 +4,5 @@
...
-def cfme_data(request):
+def cfme_data():
```
Comment (mshriver) [inline cfme/fixtures/cfme_data.py]: Wow, I don't think we're actually using this anywhere.

**Pull request merged**